django/db/backends/oracle/schema.py

```diff
@@ -42,6 +44,41 @@ def quote_value(self, value):
         else:
             return str(value)
 
+    def create_model(self, model):
+        super().create_model(model)
+        if model._meta.db_table_comment:
+            self.execute(
+                self.sql_alter_table_comment
+                % {
+                    "table": model._meta.db_table,
+                    "comment": self.quote_comment_value(model._meta.db_table_comment),
+                },
+                [],
+            )
+        for field in model._meta.local_fields:
+            if field.db_comment:
+                self.execute(
+                    self.sql_alter_column_comment
+                    % {
+                        "table": model._meta.db_table,
+                        "column": field.column,
+                        "comment": self.quote_comment_value(field.db_comment),
+                    },
+                    [],
+                )
```

Comment:
This looks the same for all backends that support this? Can we not just check the `supports_comments` flag in base class' `create_model()` method?

Reply:
mysql also supports db_comment, but there is only a difference in the sql grammar with postgresql & oracle
so it's hard to check flag `supports_comments`

Reply:
So I'm not sure it's worth having the complexity of allowing the comment to be defined inline on table creation or column addition. This is something that only MySQL/MariaDB supports, but they also support modifying the comment after creation anyway, so why not just do that consistently? We also don't want to support a hypothetical case where modification isn't possible.

If we were to keep this capability of "inline" comments I'd like to see an additional feature flag `supports_comments_inline` which can be use to toggle off the behaviour in `create_model()` and `add_field()` in the base backend. This allows us to remove everything custom from the Oracle and PostgreSQL backends other than the necessary `supports_comments` feature flag and the introspection bits.

Anyway, I'd prefer just ripping out the "inline" comments stuff.

Reply:
> Anyway, I'd prefer just ripping out the "inline" comments stuff. What's your opinion @felixxm?

Agreed, it's much simpler when done the same way on all backends.
@KimSoungRyoul Does it work for you? see 670b718afdc310d33f9459d58fd6108641a3d9a2.

Reply:
I had to restore the `supports_comments_inline` feature flag and partly also inline comments, to preserve column comments when altering column types (see also #16396).